frontend/src/components/general/MultiLevelSelector.js

```diff
@@ -351,111 +369,146 @@ function SelectedList({
 
 function ListToChooseFrom({
   className,
+  expanded,
+  isInPopup,
+  isNarrowScreen,
   isSubList,
-  parentEl,
-  parentList,
   itemsToSelectFrom,
   onClickExpand,
-  expanded,
   onClickSelect,
+  parentEl,
+  parentList,
   selected,
-  isNarrowScreen,
-  isInPopup
 }) {
   const index = isSubList ? parentList.indexOf(parentEl) : 0;
   const subListHeightCorrection = isSubList
     ? Math.min(index, Math.max(0, itemsToSelectFrom.length - parentList.length + index))
     : 0;
+
   const offset = isNarrowScreen || isInPopup ? 0 : index - subListHeightCorrection;
   const classes = useStyles({ offset: offset });
   return (
     <>
       <List
-        className={`${!isSubList && classes.list} 
-                    ${isSubList &&
+        className={`${!isSubList && classes.list}
+                    ${
+                      isSubList &&
                       (expanded === parentEl.key
                         ? isNarrowScreen || isInPopup
                           ? classes.narrowScreenSubList
                           : classes.subList
-                        : classes.hidden)} 
+                        : classes.hidden)
+                    }
                     ${className}`}
       >
-        {itemsToSelectFrom.map((item, index) => (
-          <React.Fragment key={item.key}>
-            <ListItem
-              button
-              disabled={selected.filter(s => s.key === item.key).length === 1}
-              classes={{
-                root: `${classes.listItem} 
-                        ${index == 0 && classes.firstItem} 
-                        ${isSubList && classes.subListItem} 
-                        ${isSubList &&
+        {itemsToSelectFrom.map((item, index) => {
+          // If current last item, is the last subcategory item
+          // in the last item in the outer list, then ignore our
+          // normal border styling, and paint the 1px bottom border.
+          let isFinalListItem = false;
+          if (index === itemsToSelectFrom.length - 1) {
+            const lastParentListItem = parentList && parentList[parentList.length - 1];
+            const lastParentListItemSubcategories = lastParentListItem?.subcategories;
+            const finalItem =
+              lastParentListItem?.subcategories[lastParentListItemSubcategories.length - 1];
```

Comment:
Nit: Having declared the variable above, I would use that variable here unless I'm missing a reason not to do that.
`lastParentListItemSubcategories[lastParentListItemSubcategories.length - 1]`